frontend/src/Dashboard/CrossingListPage/CrossingListItem/CrossingListItem.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import FontAwesome from 'react-fontawesome';
+import Location from './Location';
+import DateTime from './DateTime';
+import StatusToggle from './StatusToggle';
+import Dropdown from './Dropdown';
+import './CrossingListItem.css';
+
+class CrossingListItem extends React.Component {
+
+  render () {
+    var show = [];
+    switch(this.props.status) {
+      case 'Open':
+        show = this.props.dirty ? ['cancelSave'] : [];
+        break;
+      case 'Caution':
+      case 'Closed':
+        show = this.props.dirty ? ['reason', 'cancelSave'] : ['reason'];
+        break;
+      case 'Long Term Closure':
+        show = this.props.dirty ? ['reason', 'duration', 'cancelSave'] : ['reason', 'duration'];
+        break;  
+    }
+
+    return (
+      <div>
+      <div style={{height: '12px', backgroundColor: '#f5f3f7'}}></div>
+      <div className={this.props.dirty ? "DirtyBorder" : ""}>
+        <div className="CrossingListItemContainer">
+          <div className="CrossingListItemFlexContainer">
+            <div className="CrossingName">Spurlock Valley</div>
+            <Location />
+            <DateTime />
+          </div>
+          <div className="CrossingListItemFlexContainer">
+            <div className="flexitem">
+              <div className="ControlLabel">Status: {this.props.status}</div>
+              <StatusToggle status={this.props.status} />
+            </div>
+            <div className="flexitem">
+              <div className={show.includes('reason') ? "" : "hidden"}>
```

Comment:
Update this logic to not use show.includes() as a part of moving away from the [stringly typed (#7)](https://blog.codinghorror.com/new-programming-jargon/) logic here